requirements.txt

```
@@ -0,0 +1,5 @@
+huggingface-hub==0.10.1
+transformers==4.23.0
+fastcore==1.5.27
+fastdownload==0.0.7
+git+https://github.com/huggingface/diffusers.git@main#egg=diffusers
```

```suggestion
diffusers
```

Version `0.4.2` was released a few hours ago, it should work out of the box now.

---

That's great. I made the change.